server/routes/article.routes.js

```diff
@@ -41,4 +46,6 @@ router.post('/comment/:commentId/like', [signInAuth, doesLikeExistInCommentForUs
 
 router.post('/:slug/share', [signInAuth, shareArticleCheck, checkArticleExist], tryCatch(shareArticle));
 
+router.delete('/:slug', [signInAuth, checkArticleExist, checkAuthor], tryCatch(deleteArticle));
```

Comment:
From your routes, only the user can delete an article, then following RESTful conventions, the route for deleting a users article should be represented by `/users/article/: slug/`, what do you think?

Reply:
I don't think so. Every other user operation on his article in the codebase does not follow the `users/article` route. The `users` in the feature definition only refers to the fact that only a user can delete his article.

Reply:
@Chiazokam if the user is to delete an article it should be related to a user and not an article,  performing a delete user article with `/articles/: slug` won't be the best fit for deleting an article that is peculiar to a user

Reply:
Great idea but the rubrics specified so `DELETE /api/articles/:slug`